package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@kong-ui-public/i18n": "^2.2.2",
     "@kong/icons": "^1.13.0",
-    "@kong/kongponents": "^9.0.0-alpha < 9.0.0-beta",
+    "@kong/kongponents": "^9.0.0-pr.2157.f80d6769.0",
```

Comment:
This needs updating to the actual version of kongponents once PR-2157 has been merged and released.

Reply:
There will be no beta in v9 so maybe that constraint is not needed - no a blocker though